Paintroid/src/main/java/org/catrobat/paintroid/ui/ToastFactory.kt

```diff
@@ -16,35 +16,28 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
-package org.catrobat.paintroid.ui;
-
-import android.content.Context;
-import android.widget.Toast;
-
-import androidx.annotation.StringRes;
-
-public final class ToastFactory {
-	private static Toast currentToast;
-
-	private ToastFactory() {
-	}
-
-	public static Toast makeText(Context context, @StringRes int resId, int duration) {
-		if (currentToast != null) {
-			currentToast.cancel();
-		}
-
-		currentToast = Toast.makeText(context, resId, duration);
-		return currentToast;
-	}
-
-	public static Toast makeText(Context context, String msg, int duration) {
-		if (currentToast != null) {
-			currentToast.cancel();
-		}
-
-		currentToast = Toast.makeText(context, msg, duration);
-		return currentToast;
-	}
+package org.catrobat.paintroid.ui
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.widget.Toast
+import androidx.annotation.StringRes
+
+@SuppressLint("ShowToast")
+object ToastFactory {
+    private var currentToast: Toast? = null
+
+    @JvmStatic
+    fun makeText(context: Context, @StringRes resId: Int, duration: Int): Toast? {
+        currentToast?.cancel()
```

Comment:
I think its best to just make the `currentToast` a lateinit var and

check it like this 
`if (this::currentToast.isInitialized) {
            currentToast.cancel()
        }`
since it can never be null after the initial use

Reply:
done :)